omnibus/config/software/chef.rb

```diff
@@ -69,13 +69,6 @@
   block { log.info(log_key) { "" } }
   bundle "install --verbose", env: project_env
 
-  # For whatever reason, nokogiri software def deletes this (rather small) directory
```

Comment:
after upgrading nokogiri this started breaking with complaints in the `bundle show` stating it could not find mini_portile. Jenkins has built this full green so we should be ok.